gnomad/resources/grch38/reference_data.py

```diff
@@ -57,6 +57,31 @@ def _import_dbsnp(**kwargs) -> hl.Table:
     return dbsnp
 
 
+def _import_ensembl_interval(**kwargs) -> hl.Table:
+    """
+    Import and parse interval of protein-coding genes to HT.
+
+    Downloaded from Ensembl Archive for 101 & 105.
+    Only including the following fields: gene_stable_ID, chr, start, end, source_gene, gene_name, type
+    :param interval_file: str, the name of the interval file
+    """
+    ensembl = hl.import_table(
+        **kwargs,
+        min_partitions=100,
+        impute=True,
+    )
+
+    ensembl = ensembl.key_by(
+        interval=hl.locus_interval(
+            hl.literal("chr") + hl.str(ensembl.chr),
```

Comment:
I just want to confirm these files are grch38? it's just odd that the contigs don't have "chr" on them

Reply:
Yes, it is! From Ensemble archive, "Chromosome/scaffold name" is without "chr"